config/webpack/environment.js

```diff
@@ -31,6 +31,10 @@ environment.splitChunks((config) => {
           __dirname,
           '../../app/javascript/shared/components',
         ),
+        '@admin-controllers': path.resolve(
+          __dirname,
+          '../../app/javascript/admin/controllers',
+        ),
```

Comment:
(small blocking thing) not sure why this has appeared here 🤔  it's being flagged up as a duplicate key by the linter so I think this is just some weird merge thing and we can get rid
```suggestion
```

Reply:
oh good catch!